shell/edit/fleet.cattle.io.gitrepo.vue

```diff
@@ -23,7 +23,9 @@ import FleetClusterTargets from '@shell/components/fleet/FleetClusterTargets/ind
 import { toSeconds } from '@shell/utils/duration';
 import FleetGitRepoPaths from '@shell/components/fleet/FleetGitRepoPaths.vue';
 import FleetOCIStorageSecret from '@shell/components/fleet/FleetOCIStorageSecret.vue';
-import { DEFAULT_POLLING_INTERVAL, MINIMUM_POLLING_INTERVAL } from '@shell/models/fleet-application';
+
+const MINIMUM_POLLING_INTERVAL = 15;
```

Comment:
I would have probably kept these centralised in the same file, but it's definitely optional. No action here mate. You can disregard this